documentation/assemblies/assembly-configuring-olm.adoc

```diff
@@ -0,0 +1,39 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-installing-configuring-olm.adoc
+// assembly-installing-configuring-olm-rh.adoc
+
+:parent-context: {context}
+
+[id='assembly-configuring-olm-{context}']
+= Configuring {ProductName} using the {PlatformName} console
+
+After installing {ProductName} from the OperatorHub using the {PlatformName} console, create a new instance of a custom resource for the following items within the `openshift-operators` project:
+
+* service infrastructure configuration for an address space type (the example uses the standard address space type)
+* an authentication service
+* an address space plan
+* an address plan
+
+After creating the new instances of the custom resources, next:
```

Comment:
I find it a bit strange that this section is before the 'The following procedures use the example data that is provided when using the {PlatformName} console', because normally it would be the last thing to do after you've created the plans.

Reply:
@lulf So, the reasoning behind putting this information here is to summarize the whole process first. I'm afraid if this paragraph is moved after the includes, readers might miss it, because it's buried after the resource component creation procedures. I can switch it around if you think it makes more sense to do so (I wrestled a bit with initially, so I'm open to changing it if it seems confusing as is).

Reply:
@jenmalloy Ah, good point. Leave it as is then.